chaoscenter/web/config/oats.config.ts

```diff
@@ -0,0 +1,29 @@
+import { defineConfig } from '@harnessio/oats-cli';
+import reactQueryPlugin from '@harnessio/oats-plugin-react-query';
+import { mapKeys, omit } from 'lodash-es';
+
+export default defineConfig({
+  services: {
+    auth: {
+      url: 'https://raw.githubusercontent.com/litmuschaos/litmus/master/mkdocs/docs/auth/v3.0.0/auth-api.json',
```

Comment:
as for now, swagger file is manually being updated. Even after moving file somewhere else will not change it's purpose. We might have to deploy this file somewhere else where it can update automatically with BE changes. This will be worked on.
cc @Saranya-jena